src/extensions/rollup.config.ts

```diff
@@ -22,6 +22,19 @@ const config: RollupOptions = {
   ],
 };
 
+const rendererConfig: RollupOptions = {
+  input: "src/extensions/extension-renderer-api.ts",
+  output: [
+    { file: "types/extension-renderer-api.d.ts", format: "es", }
```

Comment:
What the output it produces? Isn't it should be wrapped somehow like `dtsModuleWrap({ name: "@lens/extensions" })`?

Reply:
Not sure what you mean, it's using `dtsModuleWrap({ name: "@lens/ui-extensions" })`.

Reply:
Ok, nevermind, I thought it was inside previous config :)